Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -413,6 +410,14 @@ variable {M}
 theorem lsmul_apply (r : R) (m : M) : lsmul R M r m = r • m := rfl
 #align linear_map.lsmul_apply LinearMap.lsmul_apply
 
+variable (R M Nₗ) in
+/-- A shorthand for the type of `R`-bilinear `N`-valued maps on `M`. -/
```

Comment:
```suggestion
/-- A shorthand for the type of `R`-bilinear `Nₗ`-valued maps on `M`. -/
```
Or should `Nₗ` in the `variable` line (and later) be just `N`?

Reply:
Let's use `Nₗ` for now rather than messing with the variables. I think `N` is used for semilinear maps in this file.